src/blog/lagom-1-5-0-M1.md

```diff
@@ -1,27 +1,47 @@
 ---
-title: Lagom 1.5.0-M1 released!
+title: Lagom 1.5.0-M2 released!
 date: 2018-07-25
 author_github: ignasi35
 tags: news
 summary: >
-    The Lagom team has released Lagom 1.5.0-M1
+    The Lagom team has released Lagom 1.5.0-M2
 ---
 
-We are happy to announce the availability of Lagom 1.5.0-M1. This is the first milestone towards the releasing of Lagom 1.5.0 which moves Lagom closer to Play and Akka. This release includes Play 2.7.0-M2 and starts moving towards supporting scala 2.13.
+We are happy to announce the availability of Lagom 1.5.0-M2 (unfortunately Lagom 1.5.0-M1 was DOA). This is the first milestone towards the releasing of Lagom 1.5.0 which moves Lagom closer to Play and Akka. This release includes Play 2.7.0-M2 and improves the cross-product development.
```

Comment:
I'd prefer not to say "DOA". It might not be clear to everyone what that means, and maybe it's better to be more explicit. Something like "we skipped M1 due to an issue with the release process".